fixtures/cameo/ts-100-ww.json

```diff
@@ -36,20 +36,11 @@
       "fineChannelAliases": ["Dimmer fine"],
       "capability": {
         "type": "Intensity",
-        "brightnessStart": "0%",
-        "brightnessEnd": "100%"
+        "brightnessStart": "off",
+        "brightnessEnd": "bright"
```

Comment:
I'd rather delete the brightness values here and rely on the implicit defaults.

Reply:
Done